tools/local-sensor/main.go

```diff
@@ -57,16 +58,67 @@ func createConnectionAndStartServer(fakeCentral *centralDebug.FakeService) (*grp
 	return conn, fakeCentral, closeF
 }
 
+type localSensorConfig struct {
+	Duration            time.Duration
+	CentralOutput       string
+	ShallRecordK8sInput bool
+	RecordK8sFile       string
+	ShallReplayK8sTrace bool
```

Comment:
nit: not a huge fan of prefixing these with `Shall*`. Maybe simply `ReplayK8sEnabled`?